lib/store.js

```diff
@@ -22,11 +22,38 @@
     /*
      * This method calls the cdnjs api with a search request
      */
-    findCollection: function(query) {
-      var url = link.apiSearch(query);
+    findAssets: function(query) {
+      var url = link.apiAssets(query);
+
       return this._call(url).then( function(res) {
         var data = res[1]; // res[0] is the header
-        return JSON.parse(data).results;
+        return JSON.parse(data).assets;
+      }).catch(function(err) {
+        colog.error("Could not connect to the CDNJS API");
+        throw new Error("Assets fetching error");
+      });
+    },
+
+    /*
+     * This method calls the cdnjs api with a search request
+     */
+    findCollection: function(query) {
+      var url = link.apiSearch(query);
+
+      return store.findAssets(query).then( function(res) {
+         return res;
+      }).then( function(assets) {
+         return store._call(url).then( function(res) {
+           var data = res[1]; // res[0] is the header
+           var json = JSON.parse(data).results;
+
+           // Emulate old behaviour
+           for (var i in json) {
+             json[i].assets = assets;
+           }
```

Comment:
Not sure what's going on here, haven't seen what the responses look like. Do I understand this correctly that we fetch all assets for whatever is returned from `/librarires/<query>`, then perform a search on the query and add all the found assets to every search result? This doesn't seem right to me.

Reply:
Now API has changed as you get assets only with this:
https://api.cdnjs.com/libraries/jquery?fields=assets&output=human
and it looks like:
```
{
  "assets": [
    {
      "version": "3.5.1",
      "files": [
        "jquery.js",
        "jquery.min.js",
        "jquery.min.map",
        "jquery.slim.js",
        "jquery.slim.min.js",
        "jquery.slim.min.map"
      ],
      "rawFiles": [
        "jquery.js",
        "jquery.min.js",
        "jquery.min.map",
        "jquery.slim.js",
        "jquery.slim.min.js",
        "jquery.slim.min.map"
      ],
      "sri": {
        "jquery.js": "sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc=",
        "jquery.min.js": "sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=",
        "jquery.slim.js": "sha256-DrT5NfxfbHvMHux31Lkhxg42LY6of8TaYyK50jnxRnM=",
        "jquery.slim.min.js": "sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs="
      }
    },
    {
      "version": "3.5.0",
      "files": [
        "jquery.js",
        "jquery.min.js",
        "jquery.min.map",
        "jquery.slim.js",
        "jquery.slim.min.js",
        "jquery.slim.min.map"
      ],
      "rawFiles": [
        "jquery.js",
        "jquery.min.js",
        "jquery.min.map",
        "jquery.slim.js",
        "jquery.slim.min.js",
        "jquery.slim.min.map"
      ],
      "sri": {
        "jquery.js": "sha256-r/AaFHrszJtwpe+tHyNi/XCfMxYpbsRg2Uqn0x3s2zc=",
        "jquery.min.js": "sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=",
        "jquery.slim.js": "sha256-sCexhaKpAfuqulKjtSY7V9H7QT0TCN90H+Y5NlmqOUE=",
        "jquery.slim.min.js": "sha256-MlusDLJIP1GRgLrOflUQtshyP0TwT/RHXsI1wWGnQhs="
      }
    },
   ... and continues all the version. 
```
It's not correct way to do it append it to every query but it makes this line happy
https://github.com/yannvanhalewyn/fletch/blob/01cd10d0fa0d2ae9709840f0c73ab23cf9d8f391/lib/helpers/extract.js#L23
because it's here assumed store.findMatching that there is assets.. othere way to do it would it just store assets somewhere and put it in store.findMatching-method